### PR TITLE
SmartSim documentation for Polaris

### PR DESCRIPTION
**Pull request by rickybalin**
Added a documentation page for installing SmartSim on Polaris and a link to the NekRS-ML repo for examples and tutorials.

### CODE REVIEW

File: docs/polaris/workflows/smartsim.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Note that `/path/to/` can either be a user's home or project directory.
+
+To use SmartSim in the future, simply load the same modules and source the virtual environment.
+
+Then set up the environment variables
+```
+export SMARTSIM_REDISAI=1.2.7
+export CC=cc
+export CXX=CC
+CUDA_DEPS_BASE=/soft/libraries
```
Comment (cjknight) [inline docs/polaris/workflows/smartsim.md:42]: These CUDA_ and CUDNN_ environment variables are missing `export` command.

File: docs/polaris/workflows/smartsim.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+export CUDNN_INCLUDE_DIR=$CUDNN_BASE/include/
+export LD_LIBRARY_PATH=$CUDNN_LIBRARY:$LD_LIBRARY_PATH
+```
+
+Now, install SmartSim and the GPU backend
+```
+git clone https://github.com/CrayLabs/SmartSim.git
+cd SmartSim
+pip install -e .
+TORCH_PATH=$( python -c 'import torch;print(torch.utils.cmake_prefix_path)' )
```
Comment (cjknight) [inline docs/polaris/workflows/smartsim.md:59]: TORCH_PATH and TF_PATH variables are missing `export` command.

File: docs/polaris/workflows/smartsim.md (hunk 1 above)
Comment (rickybalin) [reply docs/polaris/workflows/smartsim.md]: Thanks for pointing these out, I fixed those lines in latest commit.

File: docs/polaris/workflows/smartsim.md (hunk 2 above)
Comment (rickybalin) [reply docs/polaris/workflows/smartsim.md]: Thanks for pointing these out, I fixed those lines in latest commit.

**Review by cjknight — APPROVED** [review]

**Pull request merged**